rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere_rules_general">
+    <rules>
+        <rule id="websphere_rules_general-7Apvvd">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsTopicConnectionFactory" as="default"/>
+            </when>
+            <perform>
+                    <hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.TopicConnectionFactory.
+			</message>
+			<link href="TODO" description="JMS migration"/>
```

Comment:
I am adding TODO list to a link that would document JMS migration from websphere to JBoss